src/vm/arm/asmhelpers.S

```diff
@@ -441,7 +444,7 @@ LOCAL_LABEL(UMThunkStub_WrongAppDomain):
         ldr                 r2, [r3, #UMThunkMarshInfo__m_cbActualArgSize]
         cbz                 r2, LOCAL_LABEL(UM2MThunk_WrapperHelper_ArgumentsSetup)
 
-        add                 r0, r5, #UMThunkStub_StackArgs // Source pointer
+        add                 r0, r5, #UMThunkStub_StackArgsOffset // Source pointer
```

Comment:
Actually, I have no idea on this line. I considered ''add  r0, r5, #UMThunkStub_StackArgsSize'', and ''add  r0, r5, #UMThunkStub_StackArgsOffset''.

Could you let me know your opinion? I would be very helpful if there is any information on how ''UM2MThunk_WrapperHelper'' works.